src/dataframe.rs

```diff
@@ -23,8 +23,11 @@ use datafusion::arrow::datatypes::Schema;
 use datafusion::arrow::pyarrow::{PyArrowType, ToPyArrow};
 use datafusion::arrow::util::pretty;
 use datafusion::dataframe::DataFrame;
+use datafusion::parquet::basic::{BrotliLevel, Compression, GzipLevel, ZstdLevel};
+use datafusion::parquet::file::properties::WriterProperties;
+use datafusion::parquet::format::CompressionCodec;
```

Comment:
@mesejo Looks like this import is not needed?

Reply:
Thanks! Is fixed now!